src/Uno.UI.RuntimeTests.Engine.Library/Engine/ExternalRunner/_Private/DevServer.cs

```diff
@@ -45,7 +45,7 @@ internal sealed partial class DevServer : IAsyncDisposable
 	public static async Task<DevServer> Start(CancellationToken ct)
 	{
 #if !HAS_UNO_DEVSERVER
-		throw new NotSupportedException("Dev server is not supported on this platform.");
+		throw new NotSupportedException("Dev server has not been referenced.");
```

Comment:
Fail to see how it relates to the package upgrade

Reply:
Just output improvement, no needs to create a PR for that :)